docs/integration.rst

```diff
@@ -349,6 +349,7 @@ BigQuery Operators
 - :ref:`BigQueryIntervalCheckOperator` : Checks that the values of metrics given as SQL expressions are within a certain tolerance of the ones from days_back before.
 - :ref:`BigQueryCreateEmptyTableOperator` : Creates a new, empty table in the specified BigQuery dataset optionally with schema.
 - :ref:`BigQueryCreateExternalTableOperator` : Creates a new, external table in the dataset with the data in Google Cloud Storage.
+- :ref:`BigQueryCreateEmptyDatasetOperator` : Creates an empty BigQuery dataset.
```

Comment:
You need to add ` .. autoclass:: ` as well for this operator. Check line 359 to 365 of this file

```
.. _BigQueryCheckOperator:
 	 
 BigQueryCheckOperator	 BigQueryCheckOperator
 ^^^^^^^^^^^^^^^^^^^^^	 ^^^^^^^^^^^^^^^^^^^^^
 	 
 .. autoclass:: airflow.contrib.operators.bigquery_check_operator.BigQueryCheckOperator	 .. autoclass:: airflow.contrib.operators.bigquery_check_operator.BigQueryCheckOperator
 	 
```

Reply:
done https://github.com/apache/incubator-airflow/pull/3876/files#diff-5b9116060468557fd2efde8d21b933deR408